web/src/index.css

```diff
@@ -151,3 +151,13 @@ noscript div {
 noscript img {
   max-width: 300px;
 }
+
+@keyframes fade {
```

Comment:
Perhaps we should rename this to make it very clear what the use case is (such as "loading-icon-animation" or similar) since its going to be hard to find again if you just look at index.css :)

Reply:
Will do :)